app/components/SafeScreen.js

```diff
@@ -0,0 +1,20 @@
+import { SafeAreaView, StyleSheet, View } from "react-native";
+import Constants from "expo-constants";
+
+export const SafeScreen = ({ children, style }) => {
+  return (
+    <SafeAreaView style={[styles.screen, style]}>
```

Comment:
This is a great utility component. I would recommending using [react-native-safe-area-context](https://docs.expo.dev/versions/latest/sdk/safe-area-context/) instead of using the React Native version and having to add `Constants.statusBarHeight` to your styles. The library is much more stable and takes all devices into account, Constants can be buggy sometimes. The library also gives you access to a cool hook `useSafeAreaInsets`, [link](https://docs.expo.dev/versions/latest/sdk/safe-area-context/#hooks)

In using the library, you can do the following which will also help for the bottom notch on iPhone (be sure to delete line 15 if you use this):
```suggestion
    <SafeAreaView style={[styles.screen, style]} edges={["top", "bottom"]}>
```